app/server/schemas.py

```diff
@@ -147,6 +147,8 @@ def filter_rejected(self, data, many):
     sender_transfer_account    = fields.Nested("server.schemas.TransferAccountSchema", only=("id", "balance", "token"))
     recipient_transfer_account = fields.Nested("server.schemas.TransferAccountSchema", only=("id", "balance", "token"))
 
+    from_exchange_to_transfer_id = fields.Function(lambda obj: obj.from_exchange.to_transfer.id)
```

Comment:
this naming's a bit confusing, at first i though it was a conversion "from_exchange" "to_transfer" which didn't make sense. then i thought it was "from_exchange_to"'s "transfer", especially because the resulting variable was called `fromExchangeTransfer`, so i thought `fromExchangeTransfer.transfer_amount` was the wrong amount, that it'd be the amount exchanged from. only when i came and saw this line did i understand it

Reply:
yeah i had the same brain-ouchie. Maybe just even exchange_to_transfer_id - doesn't solve the full problem, but the word 'from' isn't really necessary.

Reply:
The reason I used `from_exchange_to_transfer` is that it actually follows the existing model relationships...

`from_exchange` = https://github.com/teamsempo/SempoBlockchain/blob/76228540b8fad295dee3938f0c8902642fe699e0/app/server/models/credit_transfer.py#L64
`to_transfer_id` = https://github.com/teamsempo/SempoBlockchain/blob/76228540b8fad295dee3938f0c8902642fe699e0/app/server/models/exchange.py#L128

Reply:
oh yeah i get that lol - it's just confusing when strung into a single variable name. The only reason it's called `from_exchange` and not just `exchange` on the model is because it would collide names with the other exhcnage relationship

Reply:
open to improved naming suggestions! 😄